Source/PagesController.swift

```diff
@@ -45,7 +52,13 @@ extension PagesController {
       currentIndex = index
       setViewControllers([viewController],
         direction: direction,
-        animated: true, completion: nil)
+        animated: true,
+        completion: {
+          [unowned self] (finish: Bool) -> Void in
```

Comment:
I'm still learning Swift but I was thinking if the `-> Void` is needed. Can you do something like this? :

```swift
completion: { [unowned self] finish in
  self.pagesDelegate?.pageViewController(self,
    setViewController: viewController,
    atPage: currentIndex)
})
```